apps/watch/src/components/NewContentPage/Quiz/Slides/Name.tsx

```
@@ -0,0 +1,67 @@
+        </div>
+        <button
+          className="mt-8 px-10 py-3 rounded-md bg-black text-white font-bold text-lg transition-opacity disabled:opacity-50 disabled:cursor-not-allowed"
+          onClick={handleSubmit}
+          disabled={!name.trim()}
+          tabIndex={0}
+        >
+          {t('Next')}
+        </button>
+      </div>
+    </div>
+  )
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)